app/views/static/payment.html.erb

```diff
@@ -0,0 +1,86 @@
+<html>
```

Comment:
Almost all of this is boilerplate stuff from the [Stripe docs](https://stripe.com/docs/payments/cards/collecting/web)

Reply:
I think the html and everything should be handled for us out of the box by rails. I think this page would have two `<head>`s if you look at it